crates/matrix-sdk/src/event_handler/context.rs

```diff
@@ -26,19 +26,19 @@ use crate::{room, Client};
 ///
 /// This trait defines the set of types that may be used as additional arguments
 /// in event handler functions after the event itself.
-pub trait EventHandlerContext: Sized {
+pub trait EventHandlerContext<E: ?Sized>: Sized {
```

Comment:
Is `E` stand for `Event` or `Error`?
I suspect it's for `E` without reading more of the code, but I suggest to use a longer name rather than a single letter here.